### PR TITLE
perl-bioperl 1.7.1

### PR DESCRIPTION
**Pull request by martin-mann**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/perl-bioperl/run_test.pl [hunk 1]
```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env perl
 
+my $CURRENT_BIOPERL_VERSION = "1.007001";
```
Comment (bgruening) [inline recipes/perl-bioperl/run_test.pl]: Nice one!

File: recipes/perl-bioperl/meta.yaml [hunk 2]
```diff
@@ -1,22 +1,24 @@
 package:
   name: perl-bioperl
-  version: "1.6.924"
+  version: 1.7.1
 
 build:
-  number: 6
-
+  number: 1
```
Comment (martin-mann) [inline recipes/perl-bioperl/meta.yaml]: Done

**Comment by martin-mann** [comment]
@bgruening could you please checkout `perl-bioperl-core` locally and check if it is loaded properly, ie. a package check for `Bio::Align::AlignI` is working? If so, something is wrong with the `run_test.pl`... I am confused...

**Comment by martin-mann** [comment]
#2917

**Comment by johanneskoester** [comment]
We have migrated our build system from Travis CI to Circle CI. If you would like to continue working on this PR, please update with latest upstream master and reopen this PR.

**Comment by mtw** [comment]
@martin-mann are you still working on this one?

**Comment by martin-mann** [comment]
not working on this since out of time.. :/

as far as I remember: main dependencies were resolved in [perl-bioperl-core](https://github.com/bioconda/bioconda-recipes/tree/master/recipes/perl-bioperl-core), since there is some kind of cyclic dependency of some packages, which makes this packages even more nasty..

I was manually adding the new dependencies one by one if not available in some conda deployment. awesome..

Best thing to do is to locally test and build until it is working. didnt do when I tried first and it was a mess..

also later separate the pull request: one/some for the dependencies, one for bioperl-core, one for bioperl.

crossing fingers...

**Comment by mtw** [comment]
OK, I'm currently pursuing a similar manual approach to get my Perl modules into bioconda. Will look into updating the bioperl package once I have my stuff up and running.

**Comment by tseemann** [comment]
Is this pull request dead?
those perl modules above are 'recommended' not 'mandatory'.

**Comment by druvus** [comment]
Bioperl 1.7.2 was added in  #10354 by @abretaud

**Pull request closed without merging**